### ISSUE TITLE
Unable to generate fields for calculating relative force deviation when using external lammps templates

### ISSUE THREAD

**Issue by xiaohe-njust** [deepmodeling/dpgen#1693]
### Bug summary

During the structural exploration stage, when using an external lammps template, the field “relative 1.0” cannot be generated for calculating the relative force deviation.Lammps files generated from non-external templates contain the field “relative 1.0”.I tried adding the “relative 1.0” field to the template file, but it didn't work.

Generated from external templates: pair_style      deepmd ../graph.000.pb ../graph.001.pb ../graph.002.pb ../graph.003.pb out_freq 100 out_file model_devi.out
Non external template generated: pair_style      deepmd ../graph.000.pb ../graph.001.pb ../graph.002.pb ../graph.003.pb  out_freq ${THERMO_FREQ} out_file model_devi.out relative 1.0 

### DP-GEN Version

0.12.1

### Input Files, Running Commands, Error Log, etc.

[BUG.zip](https://github.com/user-attachments/files/18207999/BUG.zip)

### Steps to Reproduce

dpgen run param.json machine.json

**Comment by njzjz** [comment]
Is it the same as your previous issue #1639? Note that we don't have enough developers, so if you could fix this issue by yourself, welcome to contribute to the code.

**Comment by xiaohe-njust** [comment]
> Is it the same as your previous issue [#1639](https://github.com/deepmodeling/dpgen/issues/1639)? Note that we don't have enough developers, so if you could fix this issue by yourself, welcome to contribute to the code.

Yes, same problem as before, found out today that it was due to not generating the appropriate fields, I hope this discovery is informative. We don't have the ability to modify the code.
